pytorch_lightning/core/lightning.py

```diff
@@ -905,24 +904,25 @@ def configure_apex(self, amp, model, optimizers, amp_level):
 
         return model, optimizers
 
-    def configure_optimizers(self) -> Union[
+    def configure_optimizers(self) -> Optional[Union[
```

Comment:
what if we just make the whole function optional? instead of just the args? or is that optional syntax doing that?

Reply:
or i guess if we make the method still required but optional return, then it maintains readability? on second thought i kind of like this better. that way it’s explicit in code that you used no optimizer

Reply:
Yeah, this does that - or at least, PyCharm isn't prompting me to implement the method :) - the optional thing just means you wont get a type warning if you return `None`

Reply:
^^ we can do that - currently method isn't required but happy to change that :) agree that this behaviour should be explicit